### PR TITLE
A few small issue fixes

### PR DESCRIPTION
**Pull request by noahsark769**
1. Updated mailer api to be a little better: `send_revolv_email` now accepts kwargs to specify if it should fail silently or not and it returns the same return code as `EmailMultiAlternatives.send`.
2. Made the app 404 instead of redirect for pages of projects which are drafted (#183)
3. Added taglines to projects (#180)

Also I think something about my git is screwed up or pre-commit isn't working since I had to commit some stuff twice. More commits I guess ¯\_(ツ)_/¯

@erictshen can I get a quick review of this?

### CODE REVIEW

File: revolv/lib/mailer.py-e [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import yaml
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template import Context, Template
+from revolv.base.utils import get_all_administrator_emails
+
```
Comment (erictshen) [inline revolv/lib/mailer.py-e]: I think this file got duplicated, make sure to not include it in the pull!

File: revolv/project/models.py [hunk 2]
```diff
@@ -146,6 +145,12 @@ class Project(models.Model):
         max_length=255,
         help_text='How would you like to title this project?'
     )
+    tagline = models.CharField(
+        max_length=100,
+        null=True,
+        blank=False,
+        help_text='Select a short tag line that describes this project. (No more than 100 characters.)'
```
Comment (erictshen) [inline revolv/project/models.py]: I think you need blank=True in order for it to not be required.

https://docs.djangoproject.com/en/1.7/ref/models/fields/

**Comment by erictshen** [comment]
LGTM, make sure to not include the extra mailer file and feel free to then ship it!

File: revolv/lib/mailer.py-e (hunk 1 above)
Comment (noahsark769) [inline revolv/lib/mailer.py-e]: Aw god dammit, why does it keep doing that

**Comment by noahsark769** [comment]
Sweet! Thanks man

**Pull request merged**